plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/ModelPreferences.java

```diff
@@ -70,6 +71,39 @@ public static SeparateConnectionBehavior parse(String value) {
         }
     }
     
+    public enum LineEndingNormalizationBehavior {
+        DEFAULT(ModelMessages.statement_line_ending_normalization_default, null),
+        LF(ModelMessages.statement_line_ending_normalization_unix, "\n"),
+        CRLF(ModelMessages.statement_line_ending_normalization_dos, "\r\n"),
+        CR(ModelMessages.statement_line_ending_normalization_mac, "\r"),
+        LFCR(ModelMessages.statement_line_ending_normalization_acron, "\n\r"),
+        NL(ModelMessages.statement_line_ending_normalization_ibm, "\025"),
+        AS_IS(ModelMessages.statement_line_ending_normalization_leave_as_is, null);
+        
+        private final String title;
+        private final String chars;
+        
+        LineEndingNormalizationBehavior(String title, String chars) {
+            this.title = title;
+            this.chars = chars;
+        }
+        
+        public String getTitle() {
+            return title;
+        }
+        
+        public String getChars() {
+            return chars;
+        }
+
+        /**
+         * Convert value to LineEndingNormalizationBehavior option
+         */
+        public static LineEndingNormalizationBehavior parse(String value) {

```

Comment:
I don't like that we have to pass the value itself. Can this method accept a `DBPPreferenceStore`? This way, we can rename it to `valueOf`.